packages/database/src/core/constants/schema.js

```diff
@@ -0,0 +1,4 @@
+export const SCHEMA_NAMES = {
+  PUBLIC: 'public',
+  LOG: 'log',
```

Comment:
I see you've gone singular here to match table names in Tupaia. I'm in two minds, but slightly lean towards a plural for the schema name, unless you have strong feelings?

Reply:
I really don't have a strong preference on this. Happy to follow your suggestions!